eclass/emboss.eclass

```diff
@@ -97,8 +96,15 @@ DOCS="AUTHORS ChangeLog NEWS README"
 #
 
 emboss_src_prepare() {
+	if [[ -f "${FILESDIR}"/${P}_fix-build-system.patch ]]; then
+		mv configure.{in,ac} || die
+		epatch "${FILESDIR}"/${P}_fix-build-system.patch
+	fi
+
+	[[ -n ${EBO_PATCH} ]] && epatch "${WORKDIR}"/${P}-upstream.patch
```

Comment:
Perhaps we should use $EBO_PATCH for the patch name

    [[ -n ${EBO_PATCH} ]] && epatch "${EBO_PATCH}"